test/verify/machineslib.py

```diff
@@ -1733,6 +1733,8 @@ def installWithErrorTest(dialog):
         self.browser.reload()
         self.browser.enter_page('/machines')
         self.browser.wait_visible("#create-new-vm:disabled")
+        # There are many reason why the button would be disabled, so check if it's correct one
+        self.browser.wait_attr("#create-new-vm", "testdata", "disabledVirtInstall")
```

Comment:
Can you please add back also the test for the disabled attribute?

Reply:
What test? The one on line 1735?